src/components/SellerPage/SellerPage.tsx

```diff
@@ -90,7 +90,7 @@ const ContentContainer = styled.div`
   max-width: 1200px;
   min-height: 1200px;
   @media (min-width: 900px) {
-    grid-template-columns: 1fr minmax(405px, 1fr);
+    grid-template-columns: 1fr minmax(405px, 40%);
```

Comment:
I'm not sure we want to max it at a percentage — I think we want the max width to be the max width it has to be in order to support the width of two buttons side by side

But it should be a pixel length

Reply:
My thinking is that: 

max_width = however wide it is on Figma in px

For mobile, the owner panel will be full width
For desktop, it would be max_width until it takes over more than 50% of the screen, then it would switch to mobile

Reply:
Sounds like the owner panel should just be fixed at 464px? Sinec it's the panel's gonna span the screeen above 900px screen size anyway

Reply:
ah yeah it should be fixed

Reply:
@bvillaroman how come we decided to update this to a percent? It should be fixed at 464px, and if it takes up more than 50% of the screen, then we should convert to mobile view no?